docs/ops/maths.toml

```diff
@@ -287,7 +287,7 @@ prototype = "N.B d"
 prototype_set = "N.B r s"
 short = "get degree `d` of scale/set scale root to `r`, scale to `s`, `s` is either bit mask (`s` >= 1) or scale preset (`s` < 1)"
 description = """
-Converts a degree in a user-defined equal temperament scale to a value usable by the CV outputs. Default values of `r` and  `s` are 0 and B101011010101, corresponding to C-major.
+Converts a degree in a user-defined equal temperament scale to a value usable by the CV outputs. Default values of `r` and  `s` are 0 and R101011010101, corresponding to C-major.
```

Comment:
would it be helpful to add a note explaining why `R` notation here makes more sense?

Reply:
also should probably mention this op will only use 12 lower bits?

Reply:
Done